src/gr11review/part1/Review1.java

```diff
@@ -1 +1,58 @@
 package gr11review.part1;
+import java.io.*;
+
+//Enter a certain day and month 
+//counts how many days are in the year with the information given 
+//Assume no leap years
+//Justin Ho Shue
+
+public class Review1 { 
+      
+  public static void main(String[] args) throws IOException {
+      
+    BufferedReader key = new BufferedReader(new InputStreamReader(System.in));
+    
+    int intday = 0; 
+    int intmonth = 0; 
+    int intyearDays = 0;
+  
+    //Allows the question to print on screen 
+    //Also allows people to type their answer
+    
+    System.out.println("Enter the month number: ");
```

Comment:
For input prompt, use `System.out.print` instead of `System.out.println` so that the input is read on the same line as the prompt.